fotoobo/tools/ems/get.py

```diff
@@ -44,14 +44,13 @@ def workgroups(host: str, custom: bool = False) -> Result[List[Dict[str, str]]]:
     Returns:
         Workgroups data in a list of dict with keys: name, id, total_devices
     """
-    result = Result[List[Dict[str, str]]]()
```

Comment:
I think these changes from `List[Dict[]]` to just `dict[]` would require to be put into "breaking changes" -> we need to do a major release then. (see also line 36)